src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -575,12 +555,14 @@ export const getInitialSwedishHouseValues = (
     yearOfConstruction: details.yearOfConstruction,
     isSubleted: details.isSubleted,
     extraBuildings: details.extraBuildings
-      .filter((b) => !!b.__typename)
-      .map((b) => ({
-        type: extraBuildingTypes[b.__typename!],
-        area: b.area,
-        hasWaterConnected: b.hasWaterConnected,
-      })),
+      .filter((building) => Boolean(building.__typename))
```

Comment:
I wonder where this comes from. The previous commit message says:

> "Get debugger and norwegian inbo quote to work kida"

😅 I can't figure out why it's needed..

Reply:
Yeah I don't know. So much weird and overly complex stuff going on in these files IMO 😭 But I don't know if it's worth digging in now? Probably not right?

Reply:
It feels like it's just some kind of check to see that the `building` exists? 🤔

Reply:
And this could also be removed 😀 

Tested Swedish house flow with and without extra buildings, and also created a Norwegian quote through the debugger after removing it and all of this worked fine.

Reply:
Yes, amazing! That was so confusing :D